Compilation failures since websocket commits

Ever since the recent websocket-related commits that require curl to be compiled from source (and it seems specifically in those cases), builds now fail:

```
make[1]: Entering directory '/usr/local/src/concord/core'
cc -fPIC -fPIC -std=c99 -pthread -D_XOPEN_SOURCE=600 -DLOG_USE_COLOR -I. -I/usr/local/include    -c -o cog-utils.o cog-utils.c
cc -fPIC -fPIC -std=c99 -pthread -D_XOPEN_SOURCE=600 -DLOG_USE_COLOR -I. -I/usr/local/include    -c -o io_poller.o io_poller.c
cc -fPIC -fPIC -std=c99 -pthread -D_XOPEN_SOURCE=600 -DLOG_USE_COLOR -I. -I/usr/local/include    -c -o user-agent.o user-agent.c
cc -fPIC -fPIC -std=c99 -pthread -D_XOPEN_SOURCE=600 -DLOG_USE_COLOR -I. -I/usr/local/include    -c -o websockets.o websockets.c
In file included from cog-utils.h:6,
                 from websockets.c:9:
attributes.h:2: warning: "PRINTF_LIKE" redefined
    2 | #  define PRINTF_LIKE(a, b) __attribute__((format(gnu_printf, a, b)))
      | 
In file included from logconf.h:10,
                 from websockets.h:15,
                 from websockets.c:8:
attributes.h:4: note: this is the location of the previous definition
    4 | #  define PRINTF_LIKE(a, b)
      | 
websockets.c: In function '_curl_ws_write_cb':
websockets.c:507:5: error: a label can only be part of a statement and a declaration is not a statement
  507 |     struct websockets *ws = userdata;
      |     ^~~~~~
websockets.c:508:5: error: expected expression before 'size_t'
  508 |     size_t realsize = size * nmemb;
      |     ^~~~~~
websockets.c:514:29: error: 'realsize' undeclared (first use in this function)
  514 |         if (ws->bytesleft > realsize) {
      |                             ^~~~~~~~
websockets.c:514:29: note: each undeclared identifier is reported only once for each function it appears in
make[1]: *** [<builtin>: websockets.o] Error 1
```

This is whether building curl 8.13.0 or 8.7.1 as in the recent CI commit. It isn't clear to me why 8.7.1 is being installed, but neither seems to work anyways.

The label before the declarations isn't legal so that is an issue here (I've now linked a PR to fix this).

This causes compilation error on these platforms:
- Debian 11
- Debian 10
- Rocky Linux 8.9

--- a/core/websockets.c
+++ b/core/websockets.c
@@ -502,11 +502,11 @@ default_on_ping(void *a,
 size_t
 _curl_ws_write_cb(char *ptr, size_t size, size_t nmemb, void *userdata)
 {
-again:
-
     struct websockets *ws = userdata;
-    size_t realsize = size * nmemb;
+    size_t realsize;
 
+again:
+    realsize = size * nmemb;
     if (ws->bytesleft != 0) {
         /* the amount bytes left to complete the message is bigger
              than the entire received message right now, so we

